sirepo/lib.py

```diff
@@ -103,7 +102,12 @@ def util(self):
 
 
 class Importer:
-
+    """
```

Comment:
Needs a summary string of what the function does. Make the Args: lines one line long and shorter with defaults. Here's an example: https://github.com/radiasoft/pykern/blob/master/pykern/pkconfig.py#L273

Reply:
Sure, but what do you mean 'with defaults'.

Reply:
And is "Uses LibAdapter to parse and import sim files" a sufficient summary of the class?

Reply:
A "what" is better than a "how". Using LibAdapter is "how".

`Imports a code's native files into Sirepo representation.`

Reply:
Here's an example with defaults: https://github.com/radiasoft/pykern/blob/master/pykern/pkyaml.py#L15

Reply:
Ok, clear now. thank you :)